VSTS/Tasks/SonarQubeScannerMsBuildEnd/task.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id": "4F733B42-528E-4EE3-BCBC-2744511425F7",
+    "name": "SonarQubeScannerMsBuildEnd",
+    "friendlyName": "[NEW] SonarQube Scanner for MSBuild - End Analysis",
+    "description": "Finish the analysis and upload the results to SonarQube",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkId=620063)",
```

Comment:
This blog post points to the existing article explaining the vsts tasks. I'll update this article to explain the deprecation as soon as you create the extension. You may want to have your own doc page